### PR TITLE
DYN-6783: Thumbnails for interactive guides

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

This PR addresses https://jira.autodesk.com/browse/DYN-6783:
- Display thumbnails for interactive guides in the gallery view (attached images)

The following [PR](https://github.com/DynamoDS/DynamoHome/pull/12) contains the changes in the react app to allow the correct functionality of this PR.
 
### UI Changes

image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [x] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Release Notes

- added base64 img for tour guides as resources to be used as thumbnails in the HomePage

### Reviewers

@QilongTang 

### FYIs

@Amoursol

### CODE REVIEW

File: src/Libraries/DSOffice/CSVNodes.cs [hunk 1]
```diff
@@ -21,30 +20,20 @@ public static partial class Data
         /// <search>write,text,file</search>
         public static void ExportCSV(string filePath, object[][] data)
         {
-            try
```
Comment (reddyashish) [inline src/Libraries/DSOffice/CSVNodes.cs:24]: why remove the try catch here?

Comment (dnenov) [reply src/Libraries/DSOffice/CSVNodes.cs]: reverted the rebased changes, it should be good now @reddyashish !

**Comment by QilongTang** [comment]
hi @dnenov I think you still have a few deletion change needs to be reverted in DSOffice project

**Comment by dnenov** [comment]
> hi @dnenov I think you still have a few deletion change needs to be reverted in DSOffice project

Oh boy, thank you for your patience @QilongTang .

**Review by QilongTang — APPROVED** [review]

File: src/DynamoCoreWpf/PublicAPI.Unshipped.txt [hunk 2]
```diff
@@ -4713,6 +4715,9 @@ static Dynamo.Wpf.Properties.Resources.GroupStylesCancelButtonText.get -> string
 static Dynamo.Wpf.Properties.Resources.GroupStylesSaveButtonText.get -> string
 static Dynamo.Wpf.Properties.Resources.HideClassicNodeLibrary.get -> string
 static Dynamo.Wpf.Properties.Resources.HideWiresPopupMenuItem.get -> string
+static Dynamo.Wpf.Properties.Resources.HomePageInteractiveGuideGettingStartedImg.get -> string
```
Comment (QilongTang) [inline src/DynamoCoreWpf/PublicAPI.Unshipped.txt:4718]: These are private APIs so they dont need to be declared

**Pull request merged**